tests/Fixer/ClassNotation/VisibilityRequiredFixerTest.php

```diff
@@ -811,5 +811,14 @@ public function provideFix74Cases()
             '<?php class Foo { public static Foo\Bar $foo; }',
             '<?php class Foo { static public Foo\Bar $foo; }',
         ];
+        yield [
+            '<?php class Foo { public array $foo; }',
```

Comment:
can you extend this test to contain not only "proper code", but also "code to be fix" ?